tests/tox.ini.in

```diff
@@ -205,3 +213,67 @@ disable =
     useless-else-on-loop,
     useless-super-delegation,
     wrong-import-position,
+
+[mypy]
+warn_redundant_casts = True
+warn_unused_ignores = False
```

Comment:
this should've hopefully made it more straightforward to satisfy @kgiusti 's machine